packages/circuits/circom/coordinator/non-qv/processMessages.circom

```diff
@@ -209,20 +209,20 @@ include "../../utils/trees/QuinaryGeneratePathIndices.circom";
     // Start from batchSize and decrement for process in reverse order.
     for (var i = batchSize - 1; i >= 0; i--) {
         // Process as vote type message.
-        var currentStateLeavesPathElement[stateTreeDepth][STATE_TREE_ARITY - 1];
-        var currentBallotPathElement[stateTreeDepth][STATE_TREE_ARITY - 1];
-        var currentVoteWeightsPathElement[voteOptionTreeDepth][VOTE_OPTION_TREE_ARITY - 1];
+        var computedCurrentStateLeavesPathElements[stateTreeDepth][STATE_TREE_ARITY - 1];
```

Comment:
don't we usually use the computed prefix for values calculated inside the circuit?

Reply:
I renamed it to not have name conflict. I thought "computed" prefix is something that is built inside circuit (computation, copying and etc).

Reply:
oh my bad I just saw now they are in fact computed inside the circuit